presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/file/FileHiveMetastore.java

```diff
@@ -266,9 +266,6 @@ else if (table.getTableType().equals(EXTERNAL_TABLE.name())) {
                 if (!externalFileSystem.isDirectory(externalLocation)) {
```

Comment:
> Hive metastore allows external location pointing at another managed
table's location, so allow the same in file metastore.

But there is no file metastore in Hive and so there is no `system metadata directory` there.

Is it only about creating external tables with "same" location as internal table would have? The error message is a bit misleading to me.

Reply:
> Is it only about creating external tables with "same" location as internal table would have? 

Yes

> The error message is a bit misleading to me.

fortunately it's going away